docs/subquery_network/node_operators/maximise-apy.md

```diff
@@ -0,0 +1,153 @@
+# Maximising Your APY
+
+After you have joined the SubQuery Network, completed indexing of your first project, and announced ready to the network, you might have questions on how to maximise your APY and attract as much delegation as possible.
+
+### Join the Node Operator community
+
+If you're a Node Operator on the Network, we strongly suggest that you keep up with the latest news, announcements, and tips by joining our Discord. Read more about the [Node Operator community here](../community.md#join-the-node-operator-community).
+
+## Quick Checks
+
+- We strongly recommend that you enable [Auto Reduce Over Allocation](./stake.md#automatically-reduce-over-allocation) to prevent being over allocated.
+- Ensure you have at least one flex plan enabled for each project deployment so we can send requests to you from our gateway.
+- Ensure projects are able to fetch metadata from deployment’s Query Service. An error will appear in the status column if there is an issue.
+- Ensure that you are online. `GET:{your_indexer_service_url}/healthy` should return a 200 with something like
+
+```json
+{
+  "autoReduceAllocation": true,
+  "controller": "",
+  "coordinatorVersion": "v2.2.3",
+  "indexer": "",
+  "os": "Debian GNU/Linux 10 4-CPU",
+  "proxyVersion": "v2.4.2",
+  "uptime": 8066
+}
+```
+
+## Weekly Tasks
+
+You may have already setup a Grafana dashboard to monitor metrics from your `indexer-coordinator` and `indexer-proxy` services, but there are important notifications and settings you should monitor in the [Network App](https://app.subquery.network/).
+
+We strongly suggest that you log in at the start of each new Era (as close as possible to the start - set a calendar reminder), to resolve any warnings or notifications, as well complete the following key tasks.
+
+### Weekly tasks - Claim rewards
+
+If you don't frequently claim your rewards (e.g. once a week), delegators can see this and may decide against delegating to support you. Make sure you claim your rewards frequently (e.g. once a week), and ideally as soon as they become available after the end of the past Era.
+
+Alternatively, ensure enough ETH balance is in your Controller Account so that rewards can be collected automatically. This will result in an Era Reward Collection status of “Frequent” on the [All Node Operators](https://app.subquery.network/indexer/all-indexers) dashboard meaning that your Delegators can also receive their rewards.
+
+### Weekly tasks - Check for new project versions
+
+Project creators might deploy new versions of their project codebases each time. We strongly suggest following their release notes and updating your project to match the recommended versions. In some cases you wont need to reindex, just update the project version.
```

Comment:
Good idea, added

```
Since [Project Boost affects the amount of Stake rewards distributed](../introduction/reward-distribution.md#network-inflation-rewards) for running a project, you may want to consider what project deployment version has the highest boost to maximise your Stake rewards. 
```